zebrad/Cargo.toml

```diff
@@ -235,6 +235,7 @@ hex-literal = "0.4.1"
 jsonrpc-core = "18.0.0"
 once_cell = "1.18.0"
 regex = "1.9.5"
+insta = { version = "1.31.0", features = ["json"] }
```

Comment:
Optional Nitpick: Is this dependency used anywhere?

Reply:
https://github.com/ZcashFoundation/zebra/blob/33987ed8cb040bf56d3f93cf545873873d62b758/zebrad/tests/acceptance.rs#L2676